packages/flutter_splash_tizen/bin/remove.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:io';
+import 'package:xml/xml.dart';
+
+XmlDocument? loadXMLFileSync(String path) {
+  File file = File(path);
+  if (file.existsSync()) {
+    return XmlDocument.parse(file.readAsStringSync());
+  }
+  return null;
+}
+
+void main() {
+  String tizenManifestPath = "tizen/tizen-manifest.xml";
+
+  XmlDocument? tizenManifest = loadXMLFileSync(tizenManifestPath);
+  if (tizenManifest == null) {
+    throw const FormatException("could not read tizen-manifext.xml!");
+  }
```

Comment:
Unnecessary exporting the method `loadXMLFileSync`. 

```suggestion
  File tizenManifestFile = File("tizen/tizen-manifest.xml");
  if (!file.existsSync()) {
    throw const FormatException("could not read tizen-manifext.xml!");
  }
  tizenManifest = XmlDocument.parse(tizenManifestFile.readAsStringSync());
```
```